PyZ3950/CQLParser.py

```diff
@@ -12,7 +12,7 @@
 from xml.dom.minidom import Node, parseString
 from PyZ3950.SRWDiagnostics import *
 # Don't use cStringIO as it borks Unicode (apparently)
-from StringIO import StringIO
+from io import StringIO
```

Comment:
https://stackoverflow.com/questions/3410309/what-is-the-difference-between-stringio-and-io-stringio-in-python2-7